token-lending/program/src/state/reserve.rs

```diff
@@ -877,6 +879,7 @@ impl Pack for Reserve {
             config_deposit_limit,
             config_borrow_limit,
             config_fee_receiver,
+            borrowing,
```

Comment:
TODO verify that carving a new value out of the existing padding will result in the new value defaulting to false

Reply:
hmm i did wonder if this is even needed actually
when checking for the repay instruction can't we just make sure that the data on the repay has the liquidity amount set to borrow liquidity+fees?